R/plot_carp.R

```diff
@@ -844,28 +867,17 @@ carp_path_plotly <- function(x,
           x = ~V1,
           y = ~V2,
           color = I("red"),
-          size = 3)
-
-      for (i in seq_along(plot_frame_init$ObsLabel)){
-        path_static <- path_static %>%
-          add_paths(
-            data = plot_frame_interactive[plot_frame_interactive$Obs==i,],
-            color = I("red"),
-            ids = ~Obs,
-            x = ~V1,
-            y = ~V2)
-      }
-
-      path_static %>%
-        add_text(data = plot_frame_init,
-                 x = ~V1,
-                 y = ~V2,
-                 text = ~ObsLabel,
+          size = 3) %>%
+        add_text(data = plot_frame_init_adj,
+                 x = ~x_adj,
+                 y = ~y_adj,
+                 text = ~label,
                  #size = label_size,
                  inherit = FALSE) %>%
+        plotly::layout(xaxis = list(title = "V1"), yaxis = list(title = "V2")) %>%
```

Comment:
Same comment as before.

Reply:
Same thing here about `axis[1]` and `axis[2]`

Reply:
Have updated.